### ISSUE TITLE
User-Agent breaks inline_google_font_css

### ISSUE THREAD

**Issue by Erseni** [pagespeed/ngx_pagespeed#1183]
I'm using latest pagespeed version 1.11.33.0-0 with nginx 1.9.14.

Without sending the user-agent the google font snippet is inline (the only html which is not minified):
`curl 'https://www.krankenkasse-guide.de/'`

When I send a user-agent, it is broken.
`curl 'https://www.krankenkasse-guide.de/' -H 'user-agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36'`

**Comment by morlovich** [comment]
Trying with +debug, among unhelpful message from other filters we get:

... So I think you may just need to increase pagespeed GoogleFontCssInlineMaxBytes bytes;
(4712 seems to be the current size, so maybe 5000 or something?)

(As a side note, it seems like the nginx release was missing https://github.com/pagespeed/ngx_pagespeed/commit/0a60e0ef8b81a28def38c0921b13a0f152e947eb which we do want for IE11 with nginx and native fetcher...)

**Comment by Erseni** [comment]
Thx.
Increasing MaxBytes worked.

Minifying the inlined stylesheets would be nice :-)

**Comment by jmarantz** [comment]
Indeed; confirmed that the rewrite_css filter is enabled, but the inlined font CSS  is not minified.

That might ease the pressure a little on the threshold.  Co-opting this bug for that.
